moon-store-mac-os/moon-store-mac-os/Services/BaseNetworkService.swift

```diff
@@ -19,6 +17,10 @@ private enum Constants {
 }
 
 class BaseNetworkService {
+    private lazy var errorHelper: MSErrorManager = {
```

Comment:
I think you can rename this property like `errorService`, `errorManager` or just `manager`

```suggestion
    private lazy var manager: MSErrorManager = {
```